android/app/src/main/res/menu/drawer.xml

```diff
@@ -2,11 +2,11 @@
     <group
         android:id="@+id/menu_top"
         android:checkableBehavior="single">
-        <!--<item-->
-            <!--android:id="@+id/nav_home"-->
-            <!--android:checked="true"-->
-            <!--android:icon="@drawable/ic_home_black_24dp"-->
-            <!--android:title="@string/menu_home" />-->
+        <item
```

Comment:
We need to remove the home option altogether sice that caused some issues with the whole app's UX